### PR TITLE
Improve contrail test

### PR DESCRIPTION
**Pull request by eonpatapon**

### CODE REVIEW

File: test/contrail.nix [hunk 1]
```diff
@@ -213,30 +221,102 @@ let
       virtualisation.vlans = [ 1 2 ];
       virtualisation.graphics = false;
 
+      environment.systemPackages = [
+        pkgs.vim
```
Comment (nlewo) [inline test/contrail.nix]: Could you remove them if this is only for debugging puprose?

Comment (nlewo) [reply test/contrail.nix]: Since the store is shared, you just have to install them on your host if required

File: test/contrail.nix [hunk 2]
```diff
@@ -273,6 +359,8 @@ let
         pkgs.jq
         cwPkgs.openstackClient
         contrailPkgs.tools.contrailApiCliWithExtra
+        # debug tools
+        pkgs.tcpdump
```
Comment (nlewo) [inline test/contrail.nix]: Could you remove this

Comment (eonpatapon) [reply test/contrail.nix]: Done

File: test/contrail.nix (hunk 1 above)
Comment (eonpatapon) [reply test/contrail.nix]: Done

**Pull request merged**